gluonnlp/data/transforms.py

```diff
@@ -177,9 +179,92 @@ def __init__(self):
         try:
             from nltk.tokenize.moses import MosesTokenizer
         except ImportError:
-            raise ImportError('NLTK or relevant packages are not installed. You must install NLTK '
-                              'in order to use the NLTKMosesTokenizer. You can refer to the '
-                              'official installation guide in https://www.nltk.org/install.html .')
+            warnings.warn('NLTK or relevant packages are not installed. '
+                          'Due to the LGPL 2.1+, moses has been deprecated in NLTK since 3.3.0. '
+                          'You must install NLTK <= 3.2.5 in order to use the '
+                          'NLTKMosesTokenizer. You can refer to the official '
+                          'installation guide in https://www.nltk.org/install.html .'
+                          ' Now try SacreMosesTokenizer using sacremoses ...')
+            try:
+                from sacremoses import MosesTokenizer
+            except ImportError:
+                warnings.warn('sacremoses is also not installed. '
```

Comment:
This should be an exception since the instance cannot be instantiated

Reply:
Same with the other class.

Reply:
The sacremoses is used in the script and notebook. To ensure that py2 user can run the script and notebook,  I add import nltk inside the exception block.

Reply:
I understand. Still, this is the second try and if the import still fails, the next line in constructing MosesTokenizer will fail, so I think it's better to just fail on the second import try.